src/components/gameScene/playerComp/PlayerCard.tsx

```diff
@@ -12,21 +12,23 @@ import {
   Typography,
 } from "@mui/joy";
 import { useEffect, useRef, useState } from "react";
-import defImg from '@app/assets/maxresdefault.jpg'
 import hubController from "@app/SignalR/HubController";
+import defImg from '@app/assets/maxresdefault.jpg'
 
 export default function PlayerCard({
   initialVal,
   step,
   playerName,
   scoreOfPlace,
-  indexOfPlace
+  indexOfPlace,
+  imgId
```

Comment:
Is it possible to just pass the PlayerPlaceData type instead of "destructuring" it?

Reply:
Yep, I've created type "TPlayerCardData".